spec/event_parser_spec.rb

```diff
@@ -70,12 +70,12 @@
 
   it "parses multiple events" do
     lines = [
-      "event: abc\r\n",
-      "data: def\r\n",
-      "id: 1\r\n",
-      "\r\n",
-      "data: ghi\r\n",
-      "\r\n"
+      "event: abc",
+      "data: def",
+      "id: 1",
+      "",
+      "data: ghi",
+      ""
     ]
```

Comment:
One thing that might come in handy for these is squiggly heredocs which deindent to the least indented field:

```ruby
lines = <<~EVENT.lines.map(&:chomp)
  event: abc
  data: def
  id: 1
  
  data: ghi

EVENT
# => ["event: abc", "data: def", "id: 1", "", "data: ghi", ""]
```

Reply:
The intention of this PR isn't to rewrite the entire unit test codebase, but if I were going to do so, I still wouldn't want to do this. In a test case where the exact content of each string in the list is important, I prefer to represent that unambiguously with basic syntax, rather than require the reader to play computer in order to imagine what the actual data will end up being after the transformation.